accounting/fiscality/taxes/fiscal_positions.rst

```diff
@@ -0,0 +1,110 @@
+==========================================
+Fiscal Positions (tax and account mapping)
+==========================================
+
+Default taxes and accounts are set on products and customers to create new transactions on the fly.
+However, you might have to use different taxes and record the transactions on different accounts,
+according to your customers' and providers' localizations and business types.
+
+**Fiscal Positions** allow you to create *sets of rules* to automatically adapt the taxes and the
+accounts used for a transaction.
+
+.. image:: media/fiscal-positions-intra-community.png
```

Comment:
suggestion: take 'broader' screenshots, given more context to the reader/user.
Example: 
![screenshot-test-database2 odoo com-2020 05 26-14_08_14](https://user-images.githubusercontent.com/57449423/82899077-a73eb880-9f5a-11ea-9a67-5087f6bddbd9.png)

:)

Reply:
agreed.